Source/Core/DolphinQt/Config/Graphics/GeneralWidget.cpp

```diff
@@ -88,6 +88,8 @@ void GeneralWidget::CreateWidgets()
   auto* m_options_layout = new QGridLayout();
 
   m_show_fps = new GraphicsBool(tr("Show FPS"), Config::GFX_SHOW_FPS);
+  m_show_vps = new GraphicsBool(tr("Show VPS"), Config::GFX_SHOW_VPS);
```

Comment:
This options should not be located in Graphics > General. I hard disagree with this.

Reply:
I'm thinking about the best way to go about this. 

I'm not sure, but I feel like this PR does work as a temporary fix until you do your rewrite because I did give thought into the order of the items and regrouped them by priority (top -> bottom) and stacked similar options ontop of eachother. I feel like the current one actually has less visual clutter just because it has similar options next to eachother.

But i'd be happy to move it tomorrow since I have essays for college tonight.

Reply:
I think a compromise would be keep Show FPS in General, and move the more 'advanced' stats options to Advanced - boom problem solved (though imo is a worse UX, but it meets Maylmilae's criteria

Reply:
I do not want things to get worse before they get better. So, I'll quickly summarize some routes you can take that I'll be ok with.

Bundling everything up under a global toggle would be the quickest route. Just one buttons turns everything on and off, that's it. That maintains the status quo. More controls can be added later; I'm planning to add some as part of my rewrite.

If you really want control over the appearance of the performance display in the GUI _today_, moving options to Advanced is another route.

The big route is a button for Performance Display options that opens a window with more options. Obviously this couldn't go into the checkmarks of Graphics > General, but it resolves a lot of density and noise issues and should work in Graphics > General. Plus, it would give you tons of room for more controls. This is the route I've been leaning toward personally for my own rework. The downside of this is that it's the most work; you don't need to do this. You can if you want to though!

No matter what route you choose going forward, you can always have controls in INIs. You'll have all the space in the world for any colour or arrangement options you want. That's what INIs are for.